addons/l10n_ee/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,354 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="tax_report_vat" model="account.report">
+        <field name="name">VAT Report (KMD)</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.ee"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="tax_report_cars" model="account.report.column">
+                <field name="name">Cars</field>
+                <field name="expression_label">cars</field>
```

Comment:
@jco-odoo It can not go in the same column, since they have different formatting (monetary vs integer), but I could place them on the same line. In that case, I would keep the column name as "Cars" for clarity on what to fill in there. What do you think?